express-notes-api/index.js

```diff
@@ -0,0 +1,113 @@
+const express = require('express');
+
+const app = express();
+
+const fs = require('fs');
+
+const data = require('./data.json');
+
+let dataModel = {
+};
+
+dataModel = data;
+
```

Comment:
You can leave this out and just use `data`.

Reply:
Thanks for the feedback, my code looks and works much better now.